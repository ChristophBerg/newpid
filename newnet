#!/bin/sh

exec newpid -in "$@"
