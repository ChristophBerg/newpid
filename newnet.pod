=head1 NAME

newnet - launch a command in a new PID and network namespace

=head1 SYNOPSIS

B<newnet> I<command>

=head1 DESCRIPTION

B<newnet> is a wrapper around B<netpid -in> for use in situations where only a
simple executable without options can be used, e.g. in B<sbuild>'s
B<$build_env_cmnd> setting.

=head1 OPTIONS

None.

=head1 EXAMPLE

 .sbuildrc:
 $build_env_cmnd = 'newnet';
 1;

=head1 SEE ALSO

newpid(1).

=head1 AUTHOR

Christoph Berg <myon@debian.org>
