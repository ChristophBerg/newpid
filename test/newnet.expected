PING 127.0.0.1 (127.0.0.1) 56(84) bytes of data.

--- 127.0.0.1 ping statistics ---
1 packets transmitted, 1 received, 0% packet loss, 

connect: Network is unreachable
